src/schema/posts.ts

```diff
@@ -646,6 +702,30 @@ export const typeDefs = /* GraphQL */ `
   }
 
   extend type Query {
+    """
+    Get specific squad post moderation item
+    """
+    squadPostModeration(
+      """
+      Id of the requested post moderation
+      """
+      id: ID!
+      """
+      Id of the source
+      """
+      sourceId: ID!
+    ): SquadPostModeration!
+
+    """
+    Get squad post moderations by source id
+    """
+    squadPostModerationsBySourceId(
```

Comment:
This should have pagination support? `graphorm.query` already supports it you just need to expose after/first.

Reply:
@AmarTrebinjac let me know if you need any help how to use our pagination.